libraries/default.rb

```diff
@@ -747,10 +762,10 @@ def maven_build_for(node_profile, current_deploy_root)
       # run this in a begin/rescue block and retry a build immediately
       # after failure a few times before permanently failing
       build_profiles = {
```

Comment:
This change was to support all-in-one builds. If you don't give a build profile, maven builds all jars.